B827EBFFFED61EBD.json

```diff
@@ -0,0 +1,18 @@
+{
+   "gateway_conf":{
+      "gateway_ID":"B827EBFFFED61EBD",
+      “servers":[
```

Comment:
This is not the right double quote character, so, most of the file is rendered invalid.